yarn-project/prover-client/src/mocks/test_context.ts

```diff
@@ -77,19 +80,33 @@ export class TestContext {
 
   static async new(
     logger: DebugLogger,
+    worldState: 'native' | 'legacy' = 'legacy',
```

Comment:
I'll enable some of the orchestrator tests to use the new world state implementation